docs/src/SUMMARY.md

```diff
@@ -44,6 +44,9 @@
   - [Sei](./chains/sei.md)
   - [Terra](./chains/terra.md)
 
+- [CLI](./cli/index.md)
+  - [Keys](./cli/keys.md)
+
```

Comment:
Updating docs will take another day, but let's try to merge and publish this today, since only `readme` will be included in published crate